src/modules/service-provider/api/provider.controller.ts

```diff
@@ -70,6 +70,7 @@ export class ProviderController {
     ): Promise<ServiceProviderResponse[]> {
         const roleIds: string[] = await permissions.getRoleIds();
         const serviceProviders: ServiceProvider<true>[] = [];
+        /* eslint-disable no-await-in-loop */
```

Comment:
This needs to be refactored, because rolle might not be necessary her

Reply:
ticket SPSH-1083 :) as discussed